api/services/rp_ivr_system_services/churn_users_service.py

```diff
@@ -0,0 +1,100 @@
+from api import app, db
+from google.cloud import bigquery
+from datetime import datetime
+from api.models.user_program import UserProgram
+from api.models.churned_users import ChurnedUsers
+from api.services.rapid_pro_services.update_user_group_service import (
+    UpdateUserGroupService,
+)
+from sqlalchemy import update
+
+
+class ChurnUsersService(object):
+    def __init__(self):
+        self.bigquery_client = bigquery.Client()
+        self.dataset_id = app.config["DATASET_ID"]
+
+    def fetch_users_to_mark_churned(self):
```

Comment:
I think this function name is not valid as the function is also marking the user as churned.